plugins/typescript/src/tasks/typescript.ts

```diff
@@ -0,0 +1,50 @@
+import { hookFork, waitOnExit } from '@dotcom-tool-kit/logger'
+import { Task } from '@dotcom-tool-kit/types'
+import type { TypeScriptSchema } from '@dotcom-tool-kit/types/lib/schema/typescript'
+import { fork } from 'child_process'
+
+const tscPath = require.resolve('typescript/bin/tsc')
```

Comment:
I think I marginally prefer using the original approach because 1.) I don't know if `.bin` is a stable path that npm won't break in a future update either, and 2.) it's more secure to explicitly use the package name here to avoid a case where a user somehow accidentally installs a malicious package like `typscript` which has its own `tsc` binary